src/test/java/com/google/firebase/messaging/FirebaseMessagingTest.java

```diff
@@ -1,82 +1,40 @@
-/*
```

Comment:
Can you re-add this license comment? Thanks!

Reply:
Done